Vagrantfile

```diff
@@ -14,17 +14,35 @@ Vagrant.configure(2) do |config|
 
   config.vm.provision "shell", inline: <<-SHELL
     sudo apt-get update
-    sudo apt-get install -y autoconf automake build-essential git libfreetype6-dev libpng12-dev libtool nmap pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
+    sudo apt-get install -y autoconf automake build-essential expect git libfreetype6-dev libpng12-dev libtool nmap openjdk-7-jdk openjdk-7-jre pkg-config python-all-dev python-matplotlib python-nose python-numpy python-pip python-zmq sshpass trace-cmd
```

Comment:
Can we split these lines? The usual "\" syntax should work... and we can keep the 80 cols...